addons/uh60_weapons/config/cfgAmmo.hpp

```diff
@@ -59,4 +61,34 @@ class CfgAmmo {
   	};
   };
 
+  class vtx_30mm_shells_ap : B_30mm_MP_Tracer_Green {
+    caliber=4.4;
+    explosive = 0.1;
+    hit = 170;
+    indirectHit = 60;
```

Comment:
Is this intended, ap having higher `indirectHit` than hedp and same range?

Reply:
There's an issue when it came to hitting components in the vehicles.  The higher indirectHit range made the rounds "punch" more than if they didn't have a higher hit.  This only really comes into play when the round hits a target and is otherwise not capible of injuring ground troops without multipe rounds landing within about 1 meter from a soft target

Reply:
I see, and the ace frags makes the HEDP not need a greater range?

Reply:
exactly.  The frags do most of the damage.  Landing a round directly onto someone is the best methood though, but as long as you're in the ballpark of 5-10 meters away, you'll do the job with enough rounds